packages/eslint-plugin/docs/rules/adjacent-overload-signatures.md

```diff
@@ -86,10 +86,16 @@ export function foo(sn: string | number): void;
 
 If you don't care about the general structure of the code, then you will not need this rule.
 
-## Compatibility
+## Related To
```

Comment:
Love the consistency fixes, thanks!

Do you have the bandwidth to add unit tests to this PR? I think it's great as-is and just want to make sure it stays that way.

Reply:
@JoshuaKGoldberg I'm willing to give it a try, but is there a current markdown testing unit test that I can use as a starting point?

Reply:
Actually I think this is a better approach: https://github.com/typescript-eslint/typescript-eslint/pull/4204#discussion_r755725630 (putting this into the website React code instead).

Historically the `.md` files were both a documentation text source of truth _and_ what we showed end users... but with the website, they're going to not really be what we send people to at all. So it'd be better to have the website React code auto-generate as much as it can.

Reply:
I've added a basic test that verifies the title-casing of h2 headers.

Doing more requires significant changes at this point due to inconsistencies in the docs. I'm willing to slowly pick away at that, but  would prefer to do that in separate PR's to keep the PR scope somewhat in check.